frontend/public/components/filter-toolbar.tsx

```diff
@@ -434,6 +434,7 @@ type RowFilterBase<R> = {
   items: RowFilterItem[];
   filter?: (input: FilterValue, obj: R) => boolean;
   defaultSelected?: string[];
+  selected?: string[];
```

Comment:
Looks to me that this prop is not used in `public/components/filter-toolbar.tsx`, so we should not add a new one here.

Instead I believe it's an issue in  `packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/PipelineRuns.tsx` that they use `selected` instead of `defaultSelected` their.

```suggestion
```